src/common/components/hosts/AITable.tsx

```diff
@@ -42,6 +42,7 @@ export type TableRow<R> = {
 export type ActionsResolver<R> = (obj: R) => (IAction | ISeparator)[];
 export type ExpandComponentProps<R> = {
   obj: R;
+  showNTP?: boolean;
```

Comment:
Rather call it `hideNTPStatus` and invert the logic. This will improve the readability.

Reply:
```suggestion
  hideNTPStatus?: boolean;
```